rules/windows/builtin/win_mal_creddumper.yml

```diff
@@ -18,15 +18,17 @@ logsource:
 detection:
     selection:
         EventID: 7045
-    keywords:
-        Message:
-          - '*WCE SERVICE*'
-          - '*WCESERVICE*'
-          - '*DumpSvc*'
-    quarkspwdump:
-        EventID: 16
-        HiveName: '*\AppData\Local\Temp\SAM*.dmp'
-    condition: ( selection and keywords ) or quarkspwdump
+        ServiceName:
+            - '*WCE SERVICE*'
+            - '*WCESERVICE*'
+            - '*DumpSvc*'
```

Comment:
I've commented this two lines in separate comment:

https://github.com/Neo23x0/sigma/blob/f11883966457e969a5f1f54f75814bd717cf59a4/rules/windows/builtin/win_mal_creddumper.yml#L22-L23

They covered in this rule but in different part of it:

https://github.com/Neo23x0/sigma/blob/f11883966457e969a5f1f54f75814bd717cf59a4/rules/windows/builtin/win_mal_creddumper.yml#L41-L42

This way, I suggest to delete them from this rule.
The same goes for:

https://github.com/Neo23x0/sigma/blob/f11883966457e969a5f1f54f75814bd717cf59a4/rules/windows/builtin/win_mal_creddumper.yml#L24

As it is already covered in this rule:

https://github.com/Neo23x0/sigma/blob/f11883966457e969a5f1f54f75814bd717cf59a4/rules/windows/builtin/win_mal_creddumper.yml#L48-L50

The only difference is that it's mentioned as one of the possible `ImagePath` field names. 
Which is correct — `dumpsvc` is a search pattern for `C:\Windows\System32\DumpSvc.exe`, that is executable (`ImagePath`) of the service created by `PWDumpX` tool. 
Service name would be [1][2] `PWDumpX Service`.

[1] https://image.slidesharecdn.com/zeronights2017kheirkhabarov-171118103000/95/hunting-for-credentials-dumping-in-windows-environment-54-638.jpg?cb=1511001336
[2] https://jpcertcc.github.io/ToolAnalysisResultSheet/details/PWDumpX.htm